build-system/tasks/visual-diff/index.js

```diff
@@ -48,8 +48,6 @@ const {waitUntilUsed} = require('tcp-port-used');
 let puppeteer;
 let percySnapshot;
 
-// CSS widths: iPhone: 375, Pixel: 411, Desktop: 1400.
```

Comment:
Move this comment to the yaml file for documentation.

Reply:
Done